File: abc/165/b_ans.c
#include <stdio.h>
#include <string.h>

typedef long long int lli;

int main() {
  // input
  lli X; scanf("%lld", &X);

  // compute
  lli money = 100, cnt = 0;
  while (money < X) {
    money += money / 100;
    cnt++;
  }

  // output
  printf("%lld\n", cnt);
  return 0;
}

File: abc/079/b_ans.py
def main():
    # input
    N = int(input())

    # compute
    l_0, l_1 = 2, 1
    if N == 1:
        print(l_1)
    else:
        for _ in range(N-1):
            l_i = l_0 + l_1
            l_0, l_1 = l_1, l_i
        print(l_i)

    # output


if __name__ == '__main__':
    main()
